types/chrome/test/index.ts

```diff
@@ -398,6 +398,26 @@ function testDebugger() {
 	});
 }
 
+// https://developer.chrome.com/extensions/declarativeContent
+function testDeclarativeContent() {
+    let activeIcon: ImageData = new ImageData(32, 32);
```

Comment:
```suggestion
    const activeIcon: ImageData = new ImageData(32, 32);
```
This is old DT package, not following strictly current checks, `const` is preffered now